packages/dashboard/src/app/views/myStories/index.js

```diff
@@ -100,19 +103,25 @@ function MyStories() {
     [getAuthors, setQueriedAuthors]
   );
 
+  if (!getAuthors) {
+    queryAuthorsBySearch = noop;
+  }
+
   useEffect(() => {
     queryAuthorsBySearch();
   }, [queryAuthorsBySearch]);
 
   useEffect(() => {
-    fetchStories({
-      page: page.value,
-      searchTerm: search.keyword,
-      sortDirection: sort.direction,
-      sortOption: sort.value,
-      status: filter.value,
-      author: author.filterId,
-    });
+    if (apiCallbacks?.fetchStories) {
```

Comment:
Should we really make this one optional? Seems kinda pointless to have a dashboard without any stories.

Maybe it should default to a function returning an empty array?

Reply:
Yeah, I think it makes sense, `fetchStories` shouldn't be optional.